images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -270,10 +271,15 @@ function Install-VsixExtension
     $vsEdition = (Get-ToolsetContent).visualStudio.edition
     try
     {
+        if (Test-IsWin22)
```

Comment:
Why do we need a condition if we have InstallPath param?

Reply:
In Windows 22 , the location `VSIXInstaller.exe` differs from previous versions of windows, in this regard, a condition was added to override the `InstallPath` variable.

Reply:
That's why param InstallPath is useless if we always rewrite it.

Reply:
Do you think it should just be moved to a variable?

Reply:
Yep, it should be a variable.

Reply:
I fixed it. Thanks.